Nanoleaf-Api/NanoleafClient.cs

```diff
@@ -94,5 +96,7 @@ public IStateEndpoint StateEndpoint
         public ISunsetEndpoint SunsetEndpoint => new SunsetEndpoint();
```

Comment:
I think we need to remove Sunset, GeoIP and Release enpoints from INanoleafClient, since they can be used regardless of which device you have configured. If you need one of those endpoints, you can just create a new instance at that moment.

Reply:
Rename these from "Endpoints" to "Modules".
Place the Modules in the "2 Modules" folder in a new Winleafs.Modules project.
Make the namespace be Winleafs.Modules.Api

How does that sound to you?.

Reply:
Strictly speaking they are APIs. Perhaps we should have a project solely for the Nanoleaf API and then a project called "External-APIs" or something (also in the API folder) that contains these